### ISSUE TITLE
Can not get the same file from git source with petitboot-1.8.0.tar.gz

### ISSUE THREAD

**Issue by WMC2021** [open-power/petitboot#87]
Hi,
I download the source file from git, per release note , tag 1.8.0 is based on commit 8cda0a3, 
In comparion to the released file, there are some foldes/files missing and different
```https://github.com/open-power/petitboot/releases/download/v1.8.0/petitboot-1.8.0.tar.gz```

Code from git
> bootstrap  configure.ac  COPYING  discover  docker  INSTALL  lib  m4  Makefile.am  man  po  
> README.md  test  TODO  ui  utils  version.sh

Sub folders in tar file
> ABOUT-NLS   compile       config.h.in   config.sub  
> configure.ac  depcomp   INSTALL     lib        m4           
> Makefile.in  missing  test         
> TODO  utils       ylwrap aclocal.m4  config.guess  
> config.rpath  configure   COPYING      
> discover  install-sh  ltmain.sh  Makefile.am  man          
> po       test-driver  ui    version.sh

Thanks

**Comment by jk-ozlabs** [comment]
Yes, the tarball has the automake/autoconf output included, which is fairly standard. Is there an issue with what is included there?
